src/pudl/workspace/setup.py

```diff
@@ -10,6 +13,55 @@
 logger = pudl.logging_helpers.get_logger(__name__)
 
 
+def get_settings(
```

Comment:
I do like passing in the dir instead of "test" vs "live," that seems to me to get more at what the "core parameters" actually are.

We could (and maybe should?) take it a step further and pass in `input_dir` and `output_dir` as optional params that override the env vars - which follows "normal" configuration [convention](https://docs.pydantic.dev/usage/settings/#field-value-priority) of param > env var > config file.

I know we said we don't want to shove this all into Pydantic yet, but their [settings support](https://docs.pydantic.dev/usage/settings/) does handle this overriding pretty cleanly. I think that's an argument for "follow the pattern, so when we do switch to Pydantic it is less disruptive."

Reply:
Yeah I like this idea, it will make the interface much more simple and flexible